src/EA.Weee.Web/Areas/Scheme/Mappings/ToViewModels/ViewAndTransferEvidenceViewModelMapTransfer.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace EA.Weee.Web.Areas.Scheme.Mappings.ToViewModels
+{
+    using EA.Prsd.Core;
+    using EA.Weee.Core.AatfEvidence;
+    using System;
+    using System.Collections.Generic;
+
+    public class ViewAndTransferEvidenceViewModelMapTransfer
```

Comment:
This looks like the same class as ReviewSubmittedEvidenceNotesViewModelMapTransfer so think we should create an abstract bass class that ReviewSubmittedEvidenceNotesViewModelMapTransfer and ViewAndTransferEvidenceViewModelMapTransfer should inherit from. The base class should have the properties and the child class passes the values to the abstract base class constructor.

Reply:
Done, I believe